common/src/main/java/org/apache/sedona/common/utils/GeomUtils.java

```diff
@@ -419,4 +419,23 @@ public static Geometry[] getSubGeometries(Geometry geom) {
         }
         return geometries;
     }
+
+
+    public static Geometry get3DGeom(Geometry geometry, double zValue, boolean addDefaultZValue) {
+        Coordinate[] coordinates = geometry.getCoordinates();
+        if (coordinates.length == 0) return geometry;
+
+        boolean is3d = !Double.isNaN(coordinates[0].z);
+        for(int i = 0; i < coordinates.length; i++) {
+            if(!is3d) {
+                if (addDefaultZValue) {
```

Comment:
The addDefaultZValue is not needed. Please remove the redundant check. It can be easily achieve by overloading the force3D function like this:

```
public static Geometry force3D(Geometry geometry, double zValue) {
        return GeomUtils.get3DGeom(geometry, zValue);
    }

    public static Geometry force3D(Geometry geometry) {
       return GeomUtils.get3DGeom(geometry, 0.0);
    }
```